fabrics.c

```diff
@@ -668,7 +675,7 @@ static int discover_from_json_config_file(nvme_root_t r, nvme_host_t h,
 				.subsysnqn	= subsysnqn,
 				.transport	= transport,
 				.traddr		= traddr,
-				.host_traddr	= cfg.host_traddr,
+				.host_traddr	= host_traddr,
```

Comment:
What about host_iface? Shouldn't we take care of that one too?

Reply:
> What about host_iface? Shouldn't we take care of that one too?

host_iface is permitted for tcp alone, and not fc or even rdma for that matter. That's the reason why I didn't include it here. But this should be doable. How about we include this fix (for addressing fc for now), and I will send an incremental patch later to take care of host_iface too?